utils/constants.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   // ADMIN
-  BOT_OWNER_ID: '189181051216592896', // Jeek
+  // BOT_OWNER_ID: '189181051216592896', // Jeek
+  BOT_OWNER_ID: '190098635969658881', // Steve
```

Comment:
you should change this back. If you want to use permissions for some of your work I'd suggest using the `getAdminIds` under utils/adminHelpers.js - just add your discordId to the list